packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md

```diff
@@ -208,19 +211,9 @@ export class LoginComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.widget.renderEl({
-      el: '#okta-signin-container'},
-      (res) => {
-        if (res.status === 'SUCCESS') {
-          this.signIn.loginRedirect('/', { sessionToken: res.session.token });
-          // Hide the widget
-          this.widget.hide();
-        }
-      },
-      (err) => {
-        throw err;
-      }
-    );
+    this.widget.showSignInAndRedirect().catch(err => {
```

Comment:
Wouldn't this be `.showSignInAndGetTokens()`?  Redirect should be reserved for okta-hosted situations.

Reply:
@swiftone 
I replaced .showSignInAndRedirect() to .showSignInAndGetTokens(), and I get and error:
`core.js:4352 ERROR TypeError: this.widget.showSignInAndGetTokens is not a function`...

Reply:
@swiftone 
Using `.showSignInToGetTokens()` no redirection was occurred.
And with `.showSignInAndRedirect()` application have the same behavior as before.

Reply:
I think its OK to preserve existing behavior in this case